### PR TITLE
feat(date extract): the labels regarding weeks and iso weeks are clearer

### PR DESCRIPTION
**Pull request by luc-leonard**
It is now clearer that ISO weeks start monday, while 'normal' weeks start sunday

### CODE REVIEW

**Comment by luc-leonard** [comment]
[First day of week: pas toujours le lundi  ?](https://trello.com/c/7PoT0uBL/2178-first-day-of-week-pas-toujours-le-lundi)

File: src/components/stepforms/DateExtractStepForm.vue [hunk 1]
```diff
@@ -76,13 +76,13 @@ export default class DateExtractStepForm extends BaseStepForm<DateExtractStep> {
     { info: 'dayOfWeek', label: 'day of week' },
```
Comment (davinov) [inline src/components/stepforms/DateExtractStepForm.vue:76]: Also here?

**Review by davinov — CHANGES_REQUESTED** [review]
It would be nice to add this everywhere we talk about weeks or iso weeks:
- firstDayOfPreviousWeek
- firstDayOfPreviousIsoWeek
- week
- previousWeek
- previousIsoWeek

File: src/components/stepforms/DateExtractStepForm.vue (hunk 1 above)
Comment (luc-leonard) [reply src/components/stepforms/DateExtractStepForm.vue:76]: :ok:

**Review by davinov — APPROVED** [review]
Thanks :+1:

**Pull request merged**